pkgs/by-name/jo/jol/package.nix

```
@@ -0,0 +1,59 @@
+{
+  maven,
+  lib,
+  fetchFromGitHub,
+  jre_minimal,
+  makeWrapper,
+  nix-update-script,
+}:
+maven.buildMavenPackage rec {
+  pname = "jol";
+  version = "0.17";
+
+  src = fetchFromGitHub {
+    owner = "OpenJDK";
+    repo = "jol";
+    tag = version;
+    hash = "sha256-ZJFuY2QYB8eUS3y3VRMGGwklCS93HHVkNe/dhyIx0SY=";
+  };
+
+  mvnHash = "sha256-yQfiHlAZZgINGAYVlK5JflWX3d8Axtv1Ke89S7x86G4=";
+
```

Sorry about this nitpick

```suggestion

  postPatch = ''
    substituteInPlace jol-cli/src/main/java/org/openjdk/jol/Main.java \
      --replace-fail 'Usage: jol-cli.jar' 'Usage: jol-cli'
  '';

```

---

Good catch, done